tasks/helpers/bsc-fork.ts

```diff
@@ -0,0 +1,40 @@
+import { ethers } from 'ethers';
+import { NETWORKS_RPC_URL } from '../../helper-hardhat-config';
+import { DRE, getNetworkName, isForkNetwork, sleep } from '../../helpers/misc-utils';
+import { eOtherNetwork } from '../../helpers/types';
+
+export const workaroundForBSCFork = async () => {
+  if (!isForkNetwork() || getNetworkName() != eOtherNetwork.bsc) {
+    return;
+  }
+
+  const url = NETWORKS_RPC_URL[eOtherNetwork.bsc];
+  const bscProvider = new ethers.providers.JsonRpcProvider(url);
+  let latestBlock = (await bscProvider.getBlockNumber()) + 5;
```

Comment:
I'd use const because latestBlock isn't changed here.

Reply:
it was changing ;) fixed